src/main/java/org/csanchez/jenkins/plugins/kubernetes/watch/PodStatusEventHandler.java

```diff
@@ -50,7 +50,9 @@ private String formatPodStatus(PodCondition c, String phase, StringBuilder sb) {
             // not interesting
             return "";
         }
-        String formatted = String.format("%n\tPod [%s][%s] %s", phase, c.getReason(), c.getMessage());
+        String message = c.getMessage();
+        String formatted =
+                String.format("%n\tPod [%s][%s] %s", phase, c.getReason(), message != null ? message : "No message");
```

Comment:
Unrelated, but I just saw it during testing and thought it would be ok to slip it here :)